dydx/dydxChart/dydxChart/_Graphing/DateTimeAxisFormatter.swift

```diff
@@ -120,6 +120,7 @@ import Foundation
     open func stringForValue(_ value: Double, axis _: AxisBase?) -> String {
         var result: String = ""
         if let anchor = GraphingAnchor.shared?.date {
+            // what is 2500 here?
```

Comment:
@ruixhuang any idea on this number?

Reply:
Not sure... Try changing it and see how it looks